<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1900</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="authentificationStack">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1991</width>
      <height>1200</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(222, 221, 218);</string>
    </property>
    <widget class="QWidget" name="loginPage">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QListView" name="form">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>200</y>
        <width>1000</width>
        <height>600</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
border-color: rgb(192, 191, 188);
border-radius: 50px;</string>
      </property>
     </widget>
     <widget class="QFrame" name="imageFrame">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>200</y>
        <width>551</width>
        <height>600</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(246, 245, 244);
border-color: rgb(119, 118, 123);
border-radius: 50px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QLabel" name="imageLabel1">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>70</y>
         <width>480</width>
         <height>480</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="startLabel">
      <property name="geometry">
       <rect>
        <x>1190</x>
        <y>230</y>
        <width>241</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
font: 20pt &quot;CaskaydiaCove Nerd Font Propo&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Login here</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="UsernameInput">
      <property name="geometry">
       <rect>
        <x>1080</x>
        <y>370</y>
        <width>351</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;CaskaydiaCove Nerd Font Propo&quot;;
padding-top: 10px;
padding-left: 5px;
border-radius: 20px;</string>
      </property>
      <property name="placeholderText">
       <string>Entrez votre username ici</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="PasswordInput">
      <property name="geometry">
       <rect>
        <x>1080</x>
        <y>500</y>
        <width>351</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;CaskaydiaCove Nerd Font Propo&quot;;
padding-top: 10px;
paddin-left: 5px;
border-radius: 20px;</string>
      </property>
      <property name="placeholderText">
       <string>Entrez votre mot de passe ici</string>
      </property>
     </widget>
     <widget class="QLabel" name="Label1">
      <property name="geometry">
       <rect>
        <x>1080</x>
        <y>310</y>
        <width>241</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
font: 16pt &quot;CaskaydiaCove Nerd Font Propo&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLabel" name="Label2">
      <property name="geometry">
       <rect>
        <x>1080</x>
        <y>440</y>
        <width>241</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
font: 16pt &quot;CaskaydiaCove Nerd Font Propo&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LoginButton">
      <property name="geometry">
       <rect>
        <x>1080</x>
        <y>600</y>
        <width>351</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
color: rgb(255, 255, 255);
background-color: rgb(98, 160, 234);
border-radius: 20px;
font: 16pt &quot;CaskaydiaCove Nerd Font Propo&quot;;
}
QPushButton:hover{
border: 5px solid rgb(246, 245, 244);
}</string>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SignUpButton">
      <property name="geometry">
       <rect>
        <x>1080</x>
        <y>680</y>
        <width>351</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
font: 16pt &quot;CaskaydiaCove Nerd Font Propo&quot;;
border-radius: 20px;
}
QPushButton:hover{
border: 5px solid rgb(192, 191, 188);
}</string>
      </property>
      <property name="text">
       <string>Sign Up</string>
      </property>
     </widget>
     <widget class="QLabel" name="ErrorUsername">
      <property name="geometry">
       <rect>
        <x>1080</x>
        <y>430</y>
        <width>351</width>
        <height>18</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(237, 51, 59);</string>
      </property>
      <property name="text">
       <string>Utilisateur non trouv√©</string>
      </property>
     </widget>
     <widget class="QLabel" name="ErrorPassword">
      <property name="geometry">
       <rect>
        <x>1080</x>
        <y>560</y>
        <width>351</width>
        <height>18</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(237, 51, 59);</string>
      </property>
      <property name="text">
       <string>Mot de passe incorrect</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="signupPage">
     <widget class="QListView" name="signupFORM">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>200</y>
        <width>1000</width>
        <height>600</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
border-color: rgb(192, 191, 188);
border-radius: 50px;</string>
      </property>
     </widget>
     <widget class="QFrame" name="imageFrame_2">
      <property name="geometry">
       <rect>
        <x>1080</x>
        <y>200</y>
        <width>401</width>
        <height>600</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(246, 245, 244);
border-color: rgb(119, 118, 123);
border-radius: 50px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
     </widget>
     <widget class="QLabel" name="startLabel_2">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>210</y>
        <width>241</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
font: 20pt &quot;CaskaydiaCove Nerd Font Propo&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Sign Up here</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="UsernameInputSP">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>330</y>
        <width>451</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;CaskaydiaCove Nerd Font Propo&quot;;
padding-top: 10px;
padding-left: 30px;
border-radius: 20px;</string>
      </property>
      <property name="placeholderText">
       <string>Entrez votre username ici</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="PasswordInputSP">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>450</y>
        <width>451</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;CaskaydiaCove Nerd Font Propo&quot;;
padding-left: 30px;
padding-top:10px;
border-radius: 20px;
</string>
      </property>
      <property name="placeholderText">
       <string>Entrez votre password ici</string>
      </property>
     </widget>
     <widget class="QLabel" name="Label3">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>270</y>
        <width>241</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
font: 16pt &quot;CaskaydiaCove Nerd Font Propo&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLabel" name="Label4">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>410</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
font: 16pt &quot;CaskaydiaCove Nerd Font Propo&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLabel" name="Label5">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>520</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
font: 16pt &quot;CaskaydiaCove Nerd Font Propo&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Age</string>
      </property>
     </widget>
     <widget class="QLabel" name="Label6">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>520</y>
        <width>241</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
font: 16pt &quot;CaskaydiaCove Nerd Font Propo&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>R√¥le</string>
      </property>
     </widget>
     <widget class="QPushButton" name="createBtn">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>680</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
color: rgb(255, 255, 255);
background-color: rgb(98, 160, 234);
border-radius: 20px;
font: 16pt &quot;CaskaydiaCove Nerd Font Propo&quot;;
}
QPushButton:hover{
border: 5px solid rgb(246, 245, 244);
}</string>
      </property>
      <property name="text">
       <string>Create</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cancelBtn">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>680</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
font: 16pt &quot;CaskaydiaCove Nerd Font Propo&quot;;
border-radius: 20px;
}
QPushButton:hover{
border: 5px solid rgb(192, 191, 188);
}</string>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
     <widget class="QComboBox" name="roleCombo">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>580</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;CaskaydiaCove Nerd Font Propo&quot;;
padding-left: 30px;</string>
      </property>
      <item>
       <property name="text">
        <string>Admin</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cashier</string>
       </property>
      </item>
     </widget>
     <widget class="QSpinBox" name="AgespinBox">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>580</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;CaskaydiaCove Nerd Font Propo&quot;;</string>
      </property>
     </widget>
     <widget class="QLabel" name="ErrorUsernameSP">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>390</y>
        <width>141</width>
        <height>18</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(237, 51, 59);</string>
      </property>
      <property name="text">
       <string>User not found</string>
      </property>
     </widget>
     <widget class="QLabel" name="ErrorPasswordSP">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>510</y>
        <width>191</width>
        <height>18</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(237, 51, 59);</string>
      </property>
      <property name="text">
       <string>Password incorrect</string>
      </property>
     </widget>
     <widget class="QLabel" name="ErrorAge">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>640</y>
        <width>191</width>
        <height>18</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(237, 51, 59);</string>
      </property>
      <property name="text">
       <string>Age invalide</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="HomePage"/>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
